README.md

```diff
@@ -124,6 +127,17 @@ A noter qu'il existe déjà (juste après le `populate_db`) un compte super-util
 
     $ make quality  # Will run black, isort, and flake8
 
+### Mettre à jour les dépendances
+
+La version des dépendances est consignée dans les fichiers `requirements/*.in`.
+Une fois ces fichiers modifiés, les dépendances sont figées avec l’outil
+[pip-tools](https://pypi.org/project/pip-tools/). La commande suivante permet
+de mettre à jour les fichiers de dépendances :
+
+```sh
+$ make update-deps
```

Comment:
je me demande si du coup il est clair que le process est bien:
- j'update le .in
- je lance localement un `update-deps`
- je COMMITTE les résultats. Peut etre utile à mentionner ?

Reply:
Bonne idée, ajouté !